### PR TITLE
Refactor/app/system error

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: webapp/Main.js [hunk 1]
```diff
@@ -9,14 +9,15 @@ import { BrowserRouter } from 'react-router-dom'
 import { useI18n } from '@webapp/components/hooks'
 import Routes from '@webapp/Routes'
 import * as AppState from '@webapp/app/appState'
+import { ErrorState } from '@webapp/store/app/error'
```
Comment (minotogna) [inline webapp/Main.js:12]: this should be called SystemErrorState and imported from App Store
```
import { ErrorState } from '@webapp/store/app'
```

File: webapp/Main.js [hunk 2]
```diff
@@ -9,14 +9,15 @@ import { BrowserRouter } from 'react-router-dom'
...
 import { initApp } from '@webapp/app/actions'
-import { store } from '@webapp/app/store'
+import { store } from '@webapp/store/store'
 
 const Arena = () => {
   const dispatch = useDispatch()
   const i18n = useI18n()
   const ready = useSelector(AppState.isReady)
-  const systemError = useSelector(AppState.getSystemError)
+  const systemError = useSelector(ErrorState.getSystemError)
```
Comment (minotogna) [inline webapp/Main.js:20]: use useSystemError hook

File: webapp/Routes.js [hunk 3]
```diff
@@ -15,7 +15,8 @@ import AppDialogConfirm from '@webapp/app/appDialogConfirm/appDialogConfirm'
 import AppNotificationView from '@webapp/app/appNotification/appNotificationView'
 import Guest from '@webapp/views/Guest'
 
-import { throwSystemError } from '@webapp/app/actions'
+import { ErrorActions } from '@webapp/store/app/error'
```
Comment (minotogna) [inline webapp/Routes.js:18]:
```
import { SystemErrorActions } from '@webapp/store/app'
```

File: webapp/store/app/error/state.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const appStateKey = '_app'
+export const stateKey = 'error'
+
+const isEmpty = (val) => val === undefined || val == null || val.length <= 0 || Object.values(val).length <= 0
+
+export const getSystemError = (state) =>
+  state && state[appStateKey] && state[appStateKey][stateKey] && !isEmpty(state[appStateKey][stateKey])
```
Comment (SteRiccio) [inline webapp/store/app/error/state.js]: maybe we can use R.path here and assign state[appStateKey][stateKey] to a variable

**Pull request merged**